src/components/Layout.jsx

```diff
@@ -20,6 +21,41 @@ const GlobalStyles = createGlobalStyle`
     color: ${colors.darkGrey};
     font-size: calc(1.25rem + 0.5vw);
   }
+
+  button{
+    margin:2rem;
+    padding:1rem;
+    background:${colors.white};
+    border: dashed 6px ${colors.darkPink};
+  } 
+
+  button:hover{
+    cursor:pointer;
+    border:solid 6px ${colors.darkPink};
+    border-radius:5px;
+  }
+  
+  .skip-link{
```

Comment:
As the `skip-link` is used only once (so it is not actually a global variable), I think it should be a Styled component instead.

So something like
```jsx
const SkipLink = styled.a`
// Styles of the link here
...
  &:focus {
  // Focus styles here
 }
`
```